test/circle_tests.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import numpy as np
+from src.canvas import Canvas
+from src.shapes.circle import Circle
+from src.shapes.point import Point
+
+class CirclesTest(unittest.TestCase):
```

Comment:
rename to TestCircle and you will be able to run them straight from here
same goes for all